### PR TITLE
Home page content c

### PR DESCRIPTION
**Pull request by MenarSelamet**
-Layout made
-Responsive
-Tests will be done after all the components of the home page are done
-Every branch is responsible of it's own component, please don't look at the merged componentes above or beneath, they're for testing purposes. This branch is for component C only which is the communication section :D

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -2,7 +2,11 @@
...
   theme: {
-    extend: {},
+    extend: {
+      fontFamily: {
+        poppins: ['Poppins'],
+      },
+    },
   },
   plugins: [],
 };
```
Comment (symys) [inline tailwind.config.js:12]: index.css file needs an importing
@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
you can add this

File: src/ComponentFolders/HomePage/ContentC.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import VideoCall from './Images/ContentC(VideoCall).svg';
+import VoiceCall from './Images/ContentC(VoiceCall).svg';
+import Chat from './Images/ContentC(Chat).svg';
+import Card from './Card';
+
+function ContentC() {
+  return (
+    <div className="h-5/6 w-5/6 mt-4 lg:mt-0 md:h-section flex flex-col justify-center  font-poppins pl-40 pb-16">
```
Comment (symys) [inline src/ComponentFolders/HomePage/ContentC.jsx:9]: if it possible can you center the cards, if not it's okay ✨

**Review by symys — CHANGES_REQUESTED** [review]
Great jobs, well done. Need a few changes styling 🎈

**Review by SohailMahmoud — APPROVED** [review]
Looks amazing really, I just wondered about the test file and it's clarified. Keep it up

**Review by symys — APPROVED** [review]
Well döne, looks amazing

**Pull request merged**